README.md

```diff
@@ -1,7 +1,86 @@
 # FreeCodeCampOC
+Whether you are a total beginner or well experienced, we welcome you to join us if you are interested in code pairing, enhancing your technical skills in JavaScript, JQuery, HTML, CSS, Angular.js, Node.js. and much more. This is also for anyone currently taking the Free Code Camp courses online (at [www.freecodecamp.com](www.freecodecamp.com)) hoping to connect with a local campers in Orange County, CA.
 
-Projects to collaborate on : TBA
-Let's try to get at least one commit a day
-to add: Installing Teamviewer, git
+## About Us
+To sign up for the latest events and stay up to date with the group be sure to checkout:
+* [FreeCodeCamp OC Meetup](http://www.meetup.com/Free-Code-Camp-Orange-County-CA/)
+* [Facebook](https://www.facebook.com/groups/free.code.camp.orange.county/)
 
-git: http://stackoverflow.com/questions/3903817/pull-new-updates-from-original-github-repository-into-forked-github-repository
\ No newline at end of file
+## Goals
+* Projects to collaborate on : TBA
+* Goals: Let's try to get at least one commit a day
+* Download: [TeamViewer11](http://teamviewer.com/en/), [Git](https://git-scm.com/downloads)
+
+## Getting Started with Git/GitHub
+To make sure we can all contribute to this repository, make sure to download Git. There is a good guide to setting up found here: [Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Once you have Git installed, you can run the following command to ensure it is installed:
+```
+git --version
```

Comment:
Check out the [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for instructions on how to write markdown. The part about correctly formatting code is [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting).